quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/cli/command/Build.java

```diff
@@ -94,13 +96,10 @@ public void run() {
     }
 
     private static void configureBuildClassLoader() {
-        ConfigValue ignoredArtifacts = Configuration.getCurrentBuiltTimeProperty(
-                MicroProfileConfigProvider.NS_KEYCLOAK_PREFIX + ClassLoaderOptions.IGNORE_ARTIFACTS.getKey());
-
-        if (ignoredArtifacts != null && StringUtil.isNotBlank(ignoredArtifacts.getValue())) {
-            // ignored artifacts must be set prior to starting re-augmentation
-            System.setProperty("quarkus.class-loading.removed-artifacts", ignoredArtifacts.getValue());
-        }
+        // ignored artifacts must be set prior to starting re-augmentation
+        Optional.ofNullable(Configuration.getCurrentBuiltTimeProperty(QUARKUS_REMOVED_ARTIFACTS_PROPERTY))
+                .map(ConfigValue::getValue)
+                .ifPresent(s -> System.setProperty(QUARKUS_REMOVED_ARTIFACTS_PROPERTY, s));
```

Comment:
Explanation in the: https://github.com/keycloak/keycloak/pull/25012#discussion_r1405951198